dds-test/CMakeLists.txt

```diff
@@ -70,4 +73,40 @@ else()
         )
 endif()
 
+add_executable(${PROJECT_NAME}_conversion
+    src/main.cpp
+    src/unitary/conversion.cpp
+    ../dds/src/Conversion.cpp
```

Comment:
Could you remember the rationale of this?

Reply:
I only want to test the `Conversion.cpp` functionality, as in a unitary test.